docs/basic_analysis.rst

```diff
@@ -73,7 +74,8 @@ And if the spectrum contains a continuum, then it should be subtracted first:
     >>> continuum_baseline = fit_generic_continuum(spec) #doctest:+SKIP
     >>> continuum_flux = continuum_baseline(spec.spectral_axis.value) #doctest:+SKIP
     >>> continuum = Spectrum1D(spectral_axis=spec.spectral_axis, flux=continuum_flux) #doctest:+SKIP
-    >>> c = centroid(spec-continuum, region=None) #doctest:+SKIP
+    >>> c = centroid(spec-continuum, region=None)
```

Comment:
When the doc test tries to evaluate this line, it will error on `spec-continuum`. Should this be `continuum_baseline`?